src/main/java/teammates/ui/webapi/action/SearchCommentsAction.java

```diff
@@ -20,8 +20,8 @@ protected AuthType getMinAuthLevel() {
 
     @Override
     public void checkSpecificAccessControl() {
-        // Only instructors can search for comments
-        if (!userInfo.isInstructor) {
+        // Only instructors and admins can search for comments
+        if (!userInfo.isInstructor && !userInfo.isAdmin) {
```

Comment:
Although admin is allowed here, it is actually meaningless as the `execute()` process need the corresponding instructor info.

Reply:
I just wanted to be consistent with student search actually since they’re both accessed from the same page. Any issues if I change both to instructor only?

Reply:
Up to you. Just pass down the information to you. 👍

Reply:
Those are not the same:
- Search students can be accessed through both instructor search page (instructor access) and admin search page (admin access). In fact, depending on where the API is accessed from, the result needs to be different. I put this as one of UAT round 2 issues.
- Search comments is only accessible from instructor search page and thus is only for instructor access.

Reply:
Alright, I'll revert this to the previous instructor only access then and update the access control comment